ImageAnalysis/image_analysis/analyzers/UC_GenericLightSpecCam.py

```diff
@@ -76,19 +82,30 @@ def __init__(self,
         self.optimization_central_wavelength = float(optimization_central_wavelength)
         self.optimization_bandwidth_wavelength = float(optimization_bandwidth_wavelength)
 
+        self.pointing_correction_bool = bool(pointing_correction_bool)
```

Comment:
Be careful with this, because it can give unexpected results. For example: `bool("False")` gives True (as does any string that's not empty). 
In addition, in if-statements where you use this variable, Python implicitly evaluates bool() anyway. 
If you want to enforce the type of this variable, as far as I know it'll take something like: 
```
if not isinstance(pointing_correction_bool, bool):
    raise TypeError("pointing_correction_bool should be of bool type")
```

Reply:
Ah good tip!  I ran into this issue when debugging my "configure" function for the Labview image analyzer class, but I think all the child analyzer classes I wrote previously don't explicitly check for the boolean type in their `__init__` functions.  Might be a good time to update those as well.